Foundation/XMLParser.swift

```diff
@@ -465,16 +465,18 @@ open class XMLParser : NSObject {
     }
     
     internal func _handleParseResult(_ parseResult: Int32) -> Bool {
-        return true
-        /*
         var result = true
         if parseResult != 0 {
             if parseResult != -1 {
```

Comment:
It would be simpler to write this as \:
```
if parseResult == 0 {
  return true
} else {
    if _parserError == nil {
         _parserError = NSError(domain: XMLParser.errorDomain, code: Int(parseResult))
    }
} 
return false
```

If more interpretation of `parseResult` is added in the future, it will reduce the extra levels of indentation.

Reply:
Thank you for the review. I will rewrite it.